### PR TITLE
docs: ✏️ Add debug desktop client documentation

### PR DESCRIPTION
**Pull request by calcaide**
Add how to debug desktop client documentation.

Changes on the table of contents are result of running `doc:toc`

### CODE REVIEW

File: ui/desktop/README.md [hunk 1]
```diff
@@ -176,6 +179,24 @@ Steps:
 
 TBD
 
+## Debug desktop client
+
+To debug the desktop client we recommend using node debugging client. 
+
+To start the electron app with the inspector adding a breakpoint before code starts:
+
+`$ yarn start:desktop --- --inspect-brk`.
```
Comment (ZedLi) [inline ui/desktop/README.md:188]: I've never seen `---`, only `--` what's that mean?

File: ui/desktop/README.md [hunk 2]
```diff
@@ -176,6 +179,24 @@ Steps:
 
 TBD
 
+## Debug desktop client
+
+To debug the desktop client we recommend using node debugging client.
```
Comment (ZedLi) [inline ui/desktop/README.md:184]: Could you possibly give a little more context on _when_ we would want to actually attach a debugger, maybe a scenario? I haven't come across the need yet

**Review by ZedLi — APPROVED** [review]
LGTM, just left two comments!

File: ui/desktop/README.md (hunk 1 above)
Comment (calcaide) [reply ui/desktop/README.md:188]: @ZedLi yeah, the documentation shows just `--`. I am not sure why we use the `---` in front of the `--inspect` but if you do not use the `---` does not work.
Sorry I can not answer the question.

File: ui/desktop/README.md (hunk 2 above)
Comment (calcaide) [reply ui/desktop/README.md:184]: @ZedLi Good Idea I will bring this topic to tech time.

That is correct, this instructions are just to debug the `main` process. I will update the documentation to reference that.

Absoulutley right, you do not need to use the `chrome` inspector. You can use different ones, as you mentioned you can use the one on webstorms. [More info about inspector clients here](https://nodejs.org/en/docs/guides/debugging-getting-started/#inspector-clients).

Comment (calcaide) [reply ui/desktop/README.md:184]: I would also update the documentation with and scenario that shows the need to debug the `main` process.

I used the `main process` debug to help on the fix for use `env://` on the token. Not saying you need to the debug the main process to catch that bug, but it helped me.

File: ui/desktop/README.md (hunk 1 above)
Comment (ZedLi) [reply ui/desktop/README.md:210]: Found the answer, it's for some reason `ember electron` specific, see [docs here](https://ember-electron.js.org/docs/guides/development-and-debugging#command-line-arguments).

**Review by DhariniJeeva — APPROVED** [review]

File: ui/desktop/README.md (hunk 1 above)
Comment (calcaide) [reply ui/desktop/README.md:210]: @ZedLi Awesome!!! :raised_hands:

**Pull request merged**